.zuul.yaml

```
@@ -179,9 +265,21 @@
 
 - project:
     name: infrawatch/service-telemetry-operator
-    templates:
-      - stf-crc-jobs
+    #templates:
+    #  - stf-crc-jobs
     periodic:
       jobs:
         - stf-crc-ocp_412-nightly_bundles
         - stf-crc-ocp_414-nightly_bundles
+    github-check:
+      jobs:
+        - stf-crc-ocp_412-local_build
+        - stf-crc-ocp_412-local_build-3xl
+          #- stf-crc-ocp_412-local_build-4xl
+          #- stf-crc-ocp_412-local_build-5xl
+          #- stf-crc-ocp_412-local_build-6xl
+        - stf-crc-ocp_412-local_build-index_deploy
+        - stf-crc-ocp_412-local_build-index_deploy-3xl
+          #- stf-crc-ocp_412-local_build-index_deploy-4xl
+          #- stf-crc-ocp_412-local_build-index_deploy-5xl
+          #- stf-crc-ocp_412-local_build-index_deploy-6xl
```

This should be updated once https://softwarefactory-project.io/r/c/config/+/31596 is merged

```suggestion
        - stf-crc-ocp_412-local_build
        - stf-crc-ocp_412-local_build-3xl
        - stf-crc-ocp_412-local_build-4xl
        - stf-crc-ocp_412-local_build-5xl
        - stf-crc-ocp_412-local_build-6xl
        - stf-crc-ocp_412-local_build-index_deploy
        - stf-crc-ocp_412-local_build-index_deploy-3xl
        - stf-crc-ocp_412-local_build-index_deploy-4xl
        - stf-crc-ocp_412-local_build-index_deploy-5xl
        - stf-crc-ocp_412-local_build-index_deploy-6xl
```